workspace/gh-workflow/tests/fixtures/workflow-bench.yml

```diff
@@ -61,7 +61,8 @@ jobs:
           path: bench*.txt
 
   analyze:
-    needs: build
+    needs:
+      - build
```

Comment:
Change not allowed

Reply:
`needs` is a list actually. GH allows non list elements but we can not allow that because of type safety

Reply:
Use oneormore